c/aws-c-common/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:18.04
+
+RUN apt-get -y update
+RUN apt-get -y upgrade
+
+RUN apt-get -y install git
+RUN apt-get -y install gcc-multilib
+
+RUN git clone https://github.com/gernst/sv-benchmarks.git
+RUN git clone https://github.com/gernst/aws-c-common.git
+
+WORKDIR /sv-benchmarks
+RUN git checkout aws-c-common
+WORKDIR /sv-benchmarks/c/aws-c-common
+
+RUN apt-get -y install make
+RUN ./makeall /aws-c-common /sv-benchmarks
```

Comment:
Is this already the command that builds the verification tasks? Seems a little bit weird to have this in the Dockerfile. What will one do with the container created from this image?

I would expect that the container provides everything that is necessary, and then one can run the build command inside it. Maybe it should even be created in a way such that the user provides directories with the repos to Docker. Because for example currently one cannot use this to recreate the verification tasks of a local version, or of a branch that s not the same as you are using here.